public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -393,46 +421,6 @@ export class CloudWatchDatasource extends DataSourceWithBackend<CloudWatchQuery,
     return withTeardown(queryResponse, () => this.stopQueries());
   }
 
-  private addDataLinksToLogsResponse(response: DataQueryResponse, options: DataQueryRequest<CloudWatchQuery>) {
```

Comment:
Moved to separate file for the sake of no making this file too long and splitting it a bit